dags/.airflowignore

```diff
@@ -0,0 +1 @@
+utils
```

Comment:
before i added this, the scheduler was logging an error about our slack operator not being a viable DAG. i guess we need to make sure to ignore anything in the `dags` directory that isn't a DAG.

Reply:
Thanks for pointing this out - I'll add this to the readme PR.